### PR TITLE
(dropbox) update_helper file changes

### PR DESCRIPTION
**Pull request by RedBaron2**
@AdmiringWorm @gep13 @pascalberger 

## Description

This is a condensed code fix of the dropbox update_helper file. This will allow the dropbox package to check the version with the dropbox forum and verify if that version is a stable build.
## Motivation and Context

This cleans up the rushed coding job I did previously. The code is now condensed, and still readable 😄 

## How Has this Been Tested?

Tested in Appveyor and locally, and all changes work with the given file being the only change.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package have been migrated from another repository
- [x] Fix to file(s) that will minimize system resources or time used to retrieve/push package

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this mean usually the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this mean usually the notes in the description of a package).
- [x] All files is up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by majkinetor** [comment]
BTW,  `Get-RedirectedUrl` can be reduced to `iwr -MaximumRedirection 0`

**Comment by RedBaron2** [comment]
@majkinetor Thanks. I don't remember using this in this file. If I did could you please ping it out to me.

File: automatic/dropbox/update_helper.ps1 [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
-    $HTML.Links | foreach {
-  	  if ($_.href -match "stable" ) { $stable_builds += $_.href }
-    }
-    $re_dash = '-'; $re_dashndigits = "\-\D+"; $re_t5 = 't5'; $re_abc = '[a-z]\w+'; $re_num_M = '\d+\#[M]\d+';
-    $re_dot = '.'; $re_non = ''; $re_stable = 'Stable-'; $re_build = 'Build-';
-    $stable_version = ( drpbx-builds -hrefs $stable_builds -nonstable_version $nonstable_version )
-    return $stable_version
+    $weblinks = "${env:temp}\${build}links.txt"
+    $HTML = (Invoke-WebRequest -UseBasicParsing -Uri $releases).Links`
+     | where {($_ -match $build)} | Select -First 6 | Out-File $weblinks
```
Comment (AdmiringWorm) [inline automatic/dropbox/update_helper.ps1]: why writing this out to a separate file?
Why not just set a variable containing the links instead (basically just removing `| Out-File $weblinks`)?

**Comment by RedBaron2** [comment]
Thanks @AdmiringWorm for pointing this out.
I forgot to remove that part of my diagnostic testing. It is now removed and should work without my secondary diagnostic check.

**Comment by RedBaron2** [comment]
Just noticed the failure on appveyor.i will adjust this Tuesday, and undate the PR then.

**Comment by AdmiringWorm** [comment]
> @RedBaron2 
> Just noticed the failure on appveyor.i will adjust this Tuesday, and undate the PR then.

The failure may be due to one of your functions expecting a passed string, but it's actually receiving a string array.

**Comment by RedBaron2** [comment]
@AdmiringWorm 
Appveyor seems to be happy again. Thanks for the hint it helped greatly.

**Comment by AdmiringWorm** [comment]
👍 no problem, just happened to notice it when I looked again at this PR

**Comment by AdmiringWorm** [comment]
Couldn't finish the stuff I'm working on within the timeframe I've set for myself, so I'm merging this PR now

**Pull request merged**